language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/CheckExpressionNodeContext.java

```diff
@@ -48,19 +44,15 @@ public CheckExpressionNodeContext() {
     public List<LSCompletionItem> getCompletions(BallerinaCompletionContext ctx, CheckExpressionNode node)
             throws LSCompletionException {
         List<LSCompletionItem> completionItems = new ArrayList<>();
-        NonTerminalNode nodeAtCursor = ctx.getNodeAtCursor();
         if (node.parent().kind() == SyntaxKind.ASSIGNMENT_STATEMENT
                 || node.parent().kind() == SyntaxKind.LOCAL_VAR_DECL
-                || node.parent().kind() == SyntaxKind.MODULE_VAR_DECL) {
+                || node.parent().kind() == SyntaxKind.MODULE_VAR_DECL
+                || node.parent().kind() == SyntaxKind.OBJECT_FIELD) {
```

Comment:
Not related to this PR, but just curious, why only object fields? What about record fields? 🤔

Reply:
That is coming with a separate PR :)